### ISSUE TITLE
Impossible to add image field

### ISSUE THREAD

**Issue by marioestradarosa** [thecartercenter/elmo#180]
Elmo dev env;  System Version: v5.4.2-47-g8bf4057 
 collect: 1.4.8
any form can downloaded , however if the form contains, anything aside from the basic datatypes (integer, text etc.) the resulted xml passed to Collect client doesn't have a proper form spec.
Examle: 5 questions, question # 4 is an image type ,and I get in the instance section the following:

<instance>
            <data id="1" version="3">
                <q1 />
                <q2 />
                <q3 />
                <q5 />
           </data>
 </instance>
.
.
.then at the end 

<input ref="/data/q4">
      <label ref="jr:itext('q4:label')"/>

      <hint ref="jr:itext('q4:hint')"/>

</input>

No other mention about q4 either on <bind nodeset> or  <translation> sections.

Any help would be appreciated.

**Comment by hooverlunch** [comment]
Yes this feature is under current development. If you want a stable release
of elmo, use the master branch.

On Wed, Feb 10, 2016 at 2:58 PM, Mario Estrada <notifications@github.com>
wrote:

> Elmo dev env; System Version: v5.4.2-47-g8bf4057
> collect: 1.4.8
> any form can downloaded , however if the form contains, anything aside
> from the basic datatypes (integer, text etc.) the resulted xml passed to
> Collect client doesn't have a proper form spec.
> Examle: 5 questions, question # 4 is an image type ,and I get in the
> instance section the following:
>
>
>
>
>
>
>
>
>
> .
> .
> .then at the end
>
>
>   <hint ref="jr:itext('q4:hint')"/>
>
> No other mention about q4 either on or sections.
>
> Any help would be appreciated.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thecartercenter/elmo/issues/180>.
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech

Resident, Touchstone Cohousing
touchstonecohousing.org

**Comment by marioestradarosa** [comment]
Is there any progress on adding (image capture) to Elmo?

**Comment by hooverlunch** [comment]
Yes! The latest master branch now has full support for this. Give it a try!

**Comment by marioestradarosa** [comment]
that's great!, I will give it a try !
